web/components/ChangelogModal.tsx

```diff
@@ -29,6 +29,7 @@ function buildChangelog() {
     <ChangelogSection key={35} date="2021/12/05">
       <ChangelogItem github={104}>Added 2.3 characters, weapons and artifacts.</ChangelogItem>
       <ChangelogItem github={104}>Added accuracy warning to leaked weapon pages.</ChangelogItem>
+      <ChangelogItem github={105}>Added leaked 2.4 characters and weapon.</ChangelogItem>
```

Comment:
So I made an assumption about the date for this, but it's now incorrect.

@phosphene47, should I leave adding items to the changelog to you so it can be more correct based on when you're going to deploy?

Reply:
Sure. The changelog doesn't have to be super accurate though; it can be off by a day or two. The "date" doesn't have any specific definition right now. It could be the update deployment date or PR submission date in some arbitrary timezone.

As long as the change is listed here, it's fine. If it isn't, I'll update it myself just before merging.